app/helpers/file_helper.rb

```diff
@@ -576,6 +578,39 @@ def task_submission_identifier_path_with_timestamp(type, task, timestamp)
     "#{task_submission_identifier_path(type, task)}/#{timestamp.to_s}"
   end
 
+  # Apply line wrapping to a given file.
+  # The default 160-character limit is about two lines in the rendered PDF.
+  # There is also a hard limit of 1000 characters, anything longer will be truncated to 1000 characters.
+  def line_wrap(path, width: 160)
```

Comment:
Why create an extra tmp file that has to be removed later? Why no re-write this file if needed? If this only occurs during rendering then it is working on a copy already - so there is no need to preserve the original.